### PR TITLE
Fix rule filenames during packaging

### PR DESCRIPTION
**Pull request by brokensound77**
## Issues
No issue, just small fixes to issues noticed during packaging

## Summary
Bug fixes for things needed during packaging and testing

*Will need backport to 7.13 branch*

### CODE REVIEW

File: detection_rules/rule.py [hunk 1]
```diff
@@ -386,9 +386,9 @@ def to_api_format(self, include_version=True) -> dict:
         return converted
 
     @cached
-    def sha256(self) -> str:
-        # get the hash of the API dict with the version not included, otherwise it'll always be dirty.
-        hashable_contents = self.to_api_format(include_version=False)
+    def sha256(self, include_version=False) -> str:
+        # get the hash of the API dict without the version by default, otherwise it'll always be dirty.
```
Comment (rw-access) [inline detection_rules/rule.py:390]: wait no. This causes a bad feedback loop

**Review by rw-access — CHANGES_REQUESTED** [review]
let's sync up offline

File: detection_rules/rule.py (hunk 1 above)
Comment (rw-access) [reply detection_rules/rule.py:390]: nevermind, optional is good

**Review by rw-access — APPROVED** [review]

**Comment by rw-access** [comment]
Also, if 7.13 hasn't deviated from `main` yet, then you can safely merge `main` -> `7.13` locally

**Comment by brokensound77** [comment]
> Also, if 7.13 hasn't deviated from `main` yet, then you can safely merge `main` -> `7.13` locally

It has deviated a bit, so I can cherry pick back

**Pull request merged**